src/Components/Content/Content.tsx

```diff
@@ -7,9 +7,9 @@ import ViewTaskSubmission from '../ViewTaskSubmission';
 import SingleStudentOverview from '../SingleStudentOverview/SingleStudentOverview';
```

Comment:
Create index files within each Component folder to avoid repetition of path names

Reply:
Ditto for the components below